components/organisms/o-product-details.vue

```diff
@@ -146,42 +151,35 @@ export default {
   }
 };
 </script>
+
 <style lang="scss" scoped>
+
 @import "~@storefront-ui/shared/styles/helpers/breakpoints";
 
-.product {
-  @include for-desktop {
-    display: flex;
+.o-product-details {
+  @include for-desktop{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
how does this behave when the view port grows and shrinks? Does the product info section shrink or does it get cutoff?

Reply:
It behaves some what the same way with grid also. Product info is shrinking.

![product-info-before](https://user-images.githubusercontent.com/82817616/118810906-04e5b680-b8ca-11eb-840e-9063daeef4da.png)
![product-info-after](https://user-images.githubusercontent.com/82817616/118810927-0adb9780-b8ca-11eb-8ac6-5052b924f3db.png)